FirebaseSegmentation/Sources/SEGDatabaseManager.h

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 Google
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+#import "SEGSegmentationConstants.h"
+
+/// Persist config data in sqlite database on device. Managing data read/write from/to database.
+@interface SEGDatabaseManager : NSObject
+/// Shared Singleton Instance
++ (_Nonnull instancetype)sharedInstance;
```

Comment:
Please use `NS_ASSUME_NONNULL_BEGIN/END` instead of explicit `nonnull/_Nonnull` calls.

Reply:
Done.